hasura/migrations/default/1723186809431_delete_accounts_and_reviews/up.sql

```diff
@@ -0,0 +1,7 @@
+-- Up migration to delete reviews on account deletion
+
+ALTER TABLE review
+DROP CONSTRAINT review_user_id_fkey,
+ADD CONSTRAINT review_user_id_fkey
+   FOREIGN KEY (user_id) REFERENCES "user"(id)
+   ON UPDATE CASCADE ON DELETE CASCADE;
```

Comment:
Hmm I just thought about it and we actually already just set `user_id=NULL` on account deletion, which should be okay since reviews are either anonymous or semi-anonymous (just the program is displayed). I think we can just keep that behavior instead of fully-deleting

Here's the current state:
```
  user_id INT
    REFERENCES "user"(id)
    ON UPDATE CASCADE ON DELETE SET NULL,
```